### ISSUE TITLE
Feature request. Filter option in com_plugins&view=plugins that allows to show/hide standard Joomla plugins.

### ISSUE THREAD

**Issue by hennysmafter** [joomla/joomla-cms#40990]
As the title says, it would be really useful for debugging quickly when there is such an option.

On com_plugins&view=plugins
Under the filter options in the dropdown
Having a switch that toggles all Joomla Default Plugins on or off. On by default.
Now if you find an error you can simply disable the Joomla Plugins, and then you usually end up with a small list of plugins that can be the culprit.

Last time I counted, and might have miscounted, but I reached 120+ default excellent Joomla Plugins.
I think this could be a good productivity feature.

**Comment by hennysmafter** [comment]
@brianteeman 

Yes, I mean exactly that, and I must admit I did not read your reply correctly the first time. I missed the `extension manager` part, and I was thinking am I this big of an idiot that I missed this dropdown completely on the plugin page.

But your comment is really valuable because I learned something new today because in the `extension manager` we can also set the type filter to `plugin` and then I basically have what I asked.

It would still be nice to have it by the plugin component as well.

Thank you very much!
